<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>632</width>
    <height>378</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="drawBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Draw panel</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="DrawWidget" name="widget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <zorder>controlsBox</zorder>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="controlsBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="title">
       <string>Controls</string>
      </property>
      <widget class="QSlider" name="positionXSlider">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>191</width>
         <height>22</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>191</width>
         <height>0</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QSpinBox" name="positionXSpinBox">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>40</y>
         <width>48</width>
         <height>24</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>48</width>
         <height>0</height>
        </size>
       </property>
      </widget>
      <widget class="QLabel" name="positionXLabel">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>61</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Position X</string>
       </property>
      </widget>
      <widget class="QSlider" name="positionYSlider">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>191</width>
         <height>22</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>191</width>
         <height>0</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QSpinBox" name="positionYSpinBox">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>100</y>
         <width>48</width>
         <height>24</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>48</width>
         <height>0</height>
        </size>
       </property>
      </widget>
      <widget class="QLabel" name="positionYLabel">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>61</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Position Y</string>
       </property>
      </widget>
      <widget class="QSlider" name="radiusSlider">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>191</width>
         <height>22</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>191</width>
         <height>0</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QSpinBox" name="radiusSpinBox">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>160</y>
         <width>48</width>
         <height>21</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>48</width>
         <height>0</height>
        </size>
       </property>
      </widget>
      <widget class="QLabel" name="radiusLabel">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>160</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>61</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>R</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>632</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave_configuration"/>
    <addaction name="actionLoad_configuration"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSave_configuration">
   <property name="text">
    <string>Save configuration</string>
   </property>
  </action>
  <action name="actionLoad_configuration">
   <property name="text">
    <string>Load configuration</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DrawWidget</class>
   <extends>QWidget</extends>
   <header>drawwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>positionXSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>positionXSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>515</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>583</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>positionXSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>positionXSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>583</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>515</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>positionYSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>positionYSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>583</x>
     <y>146</y>
    </hint>
    <hint type="destinationlabel">
     <x>515</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>positionYSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>positionYSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>515</x>
     <y>165</y>
    </hint>
    <hint type="destinationlabel">
     <x>583</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radiusSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>radiusSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>515</x>
     <y>225</y>
    </hint>
    <hint type="destinationlabel">
     <x>583</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radiusSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>radiusSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>583</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>515</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
